src/ui/FieldTable/FieldValue.ts

```diff
@@ -255,6 +275,12 @@ export class FieldValue extends Component {
     });
   }
 
+  private prependTextCaptionToDom(): void {
+    let elem = this.getValueContainer();
+    elem.insertBefore(this.renderTextCaption(), elem.childNodes[0]);
+    elem.style.display = 'inline-block';
```

Comment:
I would prefer if you instead add a class name to the element, then set it in the external style sheet (in sass) that this classname should be displayed inline block.